.infrastructure/helm-chart/todoapp/templates/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{.Chart.Name}}-pvc
+  namespace: {{.Values.todoapp.namespace}}
+spec:
+  volumeName: {{.Chart.Name}}-pv
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{.Values.todoapp.pvcStorageRequest}}
```

Comment:
Ensure that the `todoapp.pvcStorageRequest` value is defined in your `values.yaml` file. If it's not defined, this could lead to issues when creating the PersistentVolumeClaim. Consider providing a default value or handling the case where it might be undefined.